gke-usage-metering/src/appsscript.json

```diff
@@ -7,7 +7,7 @@
     "description": "Connect to a dataset generated by GKE Usage Metering, allowing visualization of the dataset in a DataStudio dashboard",
     "feeType": ["FREE"],
     "logoUrl": "https://cloud.google.com/images/products/logos/container-engine-logo-2x.png",
-    "name": "GKE Usage Metering",
+    "name": "GKE Usage Metering - Custom",
```

Comment:
Can we not update this?

Reply:
Please reject this pull request. I only meant to submit the first commit. Remaining was internal only.